VideoLocker/src/main/java/org/edx/mobile/task/GetCommentsListTask.java

```diff
@@ -21,6 +24,8 @@ public GetCommentsListTask(@NonNull Context context, @NonNull String responseId,
     }
 
     public Page<DiscussionComment> call() throws Exception {
-        return environment.getDiscussionAPI().getCommentsList(responseId, PAGE_SIZE, page);
+        List<String> requestedFields = Collections.singletonList("profile_image");
```

Comment:
This field should be defined as a constant somewhere, perhaps as part of an enum set for better compile-time safety checking.